.drone.yml

```diff
@@ -211,8 +235,16 @@ pipeline:
     when:
       event: [ push ]
       branch: [ master ]
-
-  translations:
+      
+  github:
```

Comment:
This section seems not related?

Reply:
@JonasFranzDEV why this section is needed here?

Reply:
@lunny @lafriks As you can see in comments: this was a merge from master to pr branch. Think a full rebase is necessary.

Reply:
@daviian What should I do for a "full rebase"?

Reply:
`git rebase master`?

Reply:
@JonasFranzDEV Usually I rebase my master to be the same as upstream/master (original gitea remote) and after checking out my pr branch I do it like @Morlinest said.